data.sql

```diff
@@ -1,5 +1,20 @@
-/* Populate database with sample data. */
+ 
+/* New data */
+
+INSERT INTO animals (id,name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES (2,'Gabumon', 'Nov 15, 2018', 2, true, 8);
```

Comment:
- [ ] As requested by the previous reviewer, Please remove this data insert as it's not specified in the project requirements.